repo/config/profile.go

```diff
@@ -24,54 +33,96 @@ var defaultServerFilters = []string{
 }
 
 // Profiles is a map holding configuration transformers. Docs are in docs/config.md
-var Profiles = map[string]Transformer{
-	"server": func(c *Config) error {
-		c.Addresses.NoAnnounce = appendSingle(c.Addresses.NoAnnounce, defaultServerFilters)
-		c.Swarm.AddrFilters = appendSingle(c.Swarm.AddrFilters, defaultServerFilters)
-		c.Discovery.MDNS.Enabled = false
-		return nil
+var Profiles = map[string]Profile{
+	"server": {
+		Description: `Disables local host discovery, recommended when
+running IPFS on machines with public IPv4 addresses.`,
+
+		Transform: func(c *Config) error {
+			c.Addresses.NoAnnounce = appendSingle(c.Addresses.NoAnnounce, defaultServerFilters)
+			c.Swarm.AddrFilters = appendSingle(c.Swarm.AddrFilters, defaultServerFilters)
+			c.Discovery.MDNS.Enabled = false
+			return nil
```

Comment:
Quick Q: shouldn't server profile also disable UPnP/NAT propagation?

Reply:
Yeah, it should